sht3x/sht3x.h

```diff
@@ -53,84 +53,176 @@ extern "C" {
  */
-int16_t sht3x_measure(void);
+int16_t sht3x_measure(sht3x_i2c_addr_t addr);
 
 /**
- * Reads out the results of a measurement that was previously started by
+ * @brief Reads out the results of a measurement that was previously started by
  * sht3x_measure(). If the measurement is still in progress, this function
  * returns an error.
  * Temperature is returned in [degree Celsius], multiplied by 1000,
  * and relative humidity in [percent relative humidity], multiplied by 1000.
  *
- * @param temperature   the address for the result of the temperature
+ * @param[in]  addr the sensor address
+ * @param[out] temperature   the address for the result of the temperature
  * measurement
- * @param humidity      the address for the result of the relative humidity
+ * @param[out] humidity      the address for the result of the relative humidity
  * measurement
+ *
  * @return              0 if the command was successful, else an error code.
  */
-int16_t sht3x_read(int32_t* temperature, int32_t* humidity);
+int16_t sht3x_read(sht3x_i2c_addr_t addr, int32_t* temperature,
+                   int32_t* humidity);
 
 /**
- * Enable or disable the SHT's low power mode
+ * @brief Enable or disable the SHT's low power mode
  *
- * @param enable_low_power_mode 1 to enable low power mode, 0 to disable
+ * @param[in] enable_low_power_mode 1 to enable low power mode, 0 to disable
  */
 void sht3x_enable_low_power_mode(uint8_t enable_low_power_mode);
 
 /**
- * Read out the serial number
+ * @brief Enable or disable the SHT's low power mode
+ *
+ * @param[in] mode power mode selector
+ */
+void sht3x_set_power_mode(sht3x_measurement_mode_t mode);
+
+/**
+ * @brief Read out the serial number
+ *
+ * @param[in]  addr the sensor address
+ * @param[out] serial    the address for the result of the serial number
  *
- * @param serial    the address for the result of the serial number
  * @return          0 if the command was successful, else an error code.
  */
-int16_t sht3x_read_serial(uint32_t* serial);
+int16_t sht3x_read_serial(sht3x_i2c_addr_t addr, uint32_t* serial);
 
 /**
- * Return the driver version
+ * @brief Return the driver version
  *
  * @return Driver version string
  */
 const char* sht3x_get_driver_version(void);
 
 /**
- * Returns the configured SHT3x address.
+ * @brief Set target temperature and humidity alert threshold
+ *
+ * @param[in] addr the sensor address
+ * @param[in] thd target alert threshold to be edited
+ * @param[in] humidity humidity threshold in 10*%RH
+ * @param[in] temperature temperature threshold in 10*%°C
+ *
+ * @return          0 if the command was successful, else an error code.
+ */
+int16_t sht3x_set_alert_thd(sht3x_i2c_addr_t addr, sht3x_alert_thd_t thd,
+                            uint16_t humidity, int16_t temperature);
+
+/**
+ * @brief Get target temperature and humidity alert threshold
  *
- * @return SHT3x_ADDRESS
+ * @param[in]  addr the sensor address
+ * @param[in] thd target alert threshold to be edited
```

Comment:
align with the other parameters

```suggestion
 * @param[in]  thd target alert threshold to be edited
```